conf/pf.conf.defaults

```diff
@@ -817,6 +807,71 @@ secure_redirect=enabled
 # If status_only_on_production is enabled, the /status page will only be available on the
 # production network. This is disabled by default
 status_only_on_production=disabled
+#
+# captive_portal.httpd_mod_qos
+#
+# Enable mod_qos
+httpd_mod_qos=disabled
+#
+# captive_portal.httpd_mod_qos_maximum_users
+#
+# The maximum connections per device
+httpd_mod_qos_maximum_connections_per_device=5
+#
+# captive_portal.httpd_mod_evasive
+#
+# Enable mod_evasive
+httpd_mod_evasive=disabled
+#
+# captive_portal.httpd_mod_evasive_white_list
+#
+# Whitelisted ip/network
+httpd_mod_evasive_white_list=127.0.0.1
+#
+# captive_portal.httpd_mod_evasive_hash_table_size
+#
+# The hash table size
+httpd_mod_evasive_hash_table_size=3097
+#
+# captive_portal.httpd_mod_evasive_page_count
+#
+# Limit the number of request on the same page
+httpd_mod_evasive_page_count=5
+#
+# captive_portal.httpd_mod_evasive_page_interval
+#
+# The intervale for the page count in second
+httpd_mod_evasive_page_interval=2
+#
+# captive_portal.httpd_mod_evasive_site_count
+#
+# Limit the number of request on the same listener
+httpd_mod_evasive_site_count=20
+#
+# captive_portal.httpd_mod_evasive_site_interval
+#
+# The intervale for the site count in second
+httpd_mod_evasive_site_interval=1
+#
+# captive_portal.httpd_mod_evasive_blocking_period
+#
+# The amount of time a client will be blocked
+httpd_mod_evasive_blocking_period=10
+#
+# captive_portal.httpd_mod_evasive_email_notify
+#
+# The email address where the notification will be sent
+httpd_mod_evasive_email_notify=root@localhost
+#
+# captive_portal.httpd_mod_evasive_system_command
+#
+# The command that will be executed when a ip becomes blacklisted
+httpd_mod_evasive_system_command="sudo ipset add portal_deny %s timeout 10 ; rm -f /usr/local/pf/logs/dos*"
```

Comment:
Let's say the default timeout is 300s but the admin want to change this timeout to 3600s so it just have to change this timeout value. Timeout is not mandatory here but it's just an example.
As i said i want in the futur to change this by a script with more logic indise, like the first time you have been catched it's for 10s but after an hundred of time it will probably be for a moment or i will probably disconnect the device and deny the radius request ....